tests/unit/FileExports/CustomerFileExportTest.php

```diff
@@ -14,6 +17,51 @@ public function getFileExportClass(): string
         return CustomerFileExport::class;
     }
 
+    public function dataProviderUserCreateByRoles()
+    {
+        $tests = [];
+        foreach (GOCustomer::VALID_ROLES as $role) {
+            $tests['valid-'.$role] = [$role, $role.'_test'];
+        }
+        foreach (GOCustomer::INVALID_ROLES as $role) {
+            $tests['invalid-'.$role] = [$role, $role.'_test'];
```

Comment:
You can make the second parameter is you expect it to be valid or not, the current assertions are hard to understand
```suggestion
            $tests['invalid-'.$role] = [$role, false];
```